### PR TITLE
Update README with names and issue links

### PR DESCRIPTION
**Pull request by mohitsuman**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,65 +1,35 @@
-# vscode-openshift-tools README
+# Visual Studio Code OpenShift Connector
 
-[![Build Status](https://travis-ci.org/redhat-developer/vscode-openshift-tools.svg?branch=master)](https://travis-ci.org/redhat-developer/vscode-openshift-tools)
+[![Build Status](https://travis-ci.org/redhat-developer/vscode-openshift-tools.svg?branch=master)](https://travis-ci.org/vscode-openshift-tools)
```
Comment (jeffmaury) [inline README.md]: What's the rationale for this change. The second URL is invalid

Comment (mohitsuman) [reply README.md]: Fixed the  invalid url. That should not have been changed.
